ReactWindows/ReactNative.Net46/Views/TextInput/PlaceholderAdorner.cs

```diff
@@ -0,0 +1,1104 @@
+﻿using System;
```

Comment:
Implementation of an adorner that is used by the Net46 TextInput control to show placeholder text and to set the placeholder text color... Other attached properties are provided to style the placeholder text's FontStyle, FontWeight, FontStretch, and FontFamily.